File: src/Observer/SecurityMonitor.java
package Observer;

public class SecurityMonitor implements Observer {

    private Login login;
    public SecurityMonitor(Login login) {
        this.login=login;
        login.attach(this);
    }

    @Override
    public void update(Observable observable) {
    int statut = login.getStatus();
        if ( statut == Login.LOGIN_WRONG_PASS ) {
            System.out.println( "sending mail to sysadmin");
        }else{
            System.out.println( "Login OK...");

        }
    }
}

File: Scripts/observer/SecurityMonitor.php
<?php
require_once('Observer.php');
require_once('Login.php');

class SecurityMonitor implements Observer {

	function __construct(Login $login) {
		$this->login = $login;
		$login->attach($this);
	}
	
	function update(Observable $observable) {
		$status = $observable->getStatus();
		if ( $status == Login::LOGIN_WRONG_PASS ) {
			echo __CLASS__.": sending mail to sysadmin<br/>";
		}else{
			echo __CLASS__.": Login OK...<br/>";
		}
	}
}
?>